mvnw

```diff
@@ -256,4 +256,4 @@ printf %s\\n "$distributionUrl" >"$TMP_DOWNLOAD_DIR/$distributionUrlNameMain/mvn
 mv -- "$TMP_DOWNLOAD_DIR/$distributionUrlNameMain" "$MAVEN_HOME" || [ -d "$MAVEN_HOME" ] || die "fail to move MAVEN_HOME"
 
 clean || :
-exec_maven "$@"
```

Comment:
here it is missing line ending:

- https://docs.github.com/en/get-started/git-basics/configuring-git-to-handle-line-endings
- https://stackoverflow.com/questions/10418975/how-to-change-line-ending-settings
- https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/

Reply:
@nrmancuso suggested in the past to supplement this into dedicated PR. this seems a good idea.

Reply:
- https://github.com/checkstyle/checkstyle/pull/17144